src/components/home/StorySection/StorySection.tsx

```diff
@@ -0,0 +1,53 @@
+import React, {Component} from 'react';
+import Story from './Story';
+import './StorySection.css';
+
+const storyData: {name: string, quote: string, picUrl: string}[] = [
+    {
+        "name": "Li M.",
+        "quote": "Lorem ipsum dolor sit amet consectetur. Aliquam sed egestas enim porttitor tristique id lectus.",
+        "picUrl": require("./../../../assets/home-page/placeholder-image.png")
```

Comment:
Shall we use relative path for images or absolute path? what about /asssets/images/home/placeholder.png?